staff_features/resources/step_definitions/resource_tree_navigate.rb

```diff
@@ -0,0 +1,141 @@
+  visit "#{STAFF_URL}/resources/new"
+
+  fill_in 'resource_title_', with: "Resource #{@uuid}"
+  fill_in 'resource_id_0_', with: "Resource #{@uuid}"
+  select 'Class', from: 'resource_level_'
+  element = find('#resource_lang_materials__0__language_and_script__language_')
+  element.send_keys('AU')
+  element.send_keys(:tab)
+
+  select 'Single', from: 'resource_dates__0__date_type_'
+  fill_in 'resource_dates__0__begin_', with: '2024'
+
+  fill_in 'resource_extents__0__number_', with: '10'
+  select 'Cassettes', from: 'resource_extents__0__extent_type_'
+
+  element = find('#resource_finding_aid_language_')
+  element.send_keys('ENG')
+  element.send_keys(:tab)
+
+  element = find('#resource_finding_aid_script_')
+  element.send_keys('Latin')
+  element.send_keys(:tab)
+
+  find('button', text: 'Save Resource', match: :first).click
+
+  wait_for_ajax
+  expect(page).to have_text "Resource Resource #{@uuid} created"
+
+  url_parts = current_url.split('/')
+  url_parts.pop
+  @resource_id = url_parts.pop
+
+  click_on 'Add Child'
+  wait_for_ajax
+  fill_in 'Title', with: "Archival Object 1 #{@uuid}"
+  select 'Class', from: 'Level of Description'
+  click_on 'Save'
+  expect(page).to have_text "Archival Object Archival Object 1 #{@uuid} on Resource Resource #{@uuid} created"
+
+  click_on 'Add Child'
+  wait_for_ajax
+  fill_in 'Title', with: "Archival Object 2 #{@uuid}"
+  select 'Class', from: 'Level of Description'
+  click_on 'Save'
+  expect(page).to have_text "Archival Object Archival Object 2 #{@uuid} created as child of Archival Object 1 #{@uuid} on Resource Resource #{@uuid}"
+  wait_for_ajax
+end
+
+Given 'the Resource is opened in edit mode' do
+  visit "#{STAFF_URL}/resources/#{@resource_id}/edit"
+
+  wait_for_ajax
+end
+
+When 'the user opens the Resource in edit mode' do
+  visit "#{STAFF_URL}/resources/#{@resource_id}/edit"
+
+  wait_for_ajax
+end
+
+Then 'the Resource is displayed as the top level of the navigation tree' do
+  rows = all('#tree-container .table .table-row', visible: :all)
+
+  expect(rows.length).to eq 4
+  expect(rows[0].text).to include "Resource #{@uuid}"
+end
+
+Then 'the tree includes the two Archival Objects' do
+  rows = all('#tree-container .table .table-row')
+
+  expect(rows.length).to eq 2
+  expect(rows[2].text).to include "Archival Object 1 #{@uuid}"
+end
+
+Then 'the Resource is highlighted in the tree' do
+  rows = all('#tree-container .table .table-row')
+
+  expect(rows.length).to eq 2
+  expect(rows[0].text).to include "Resource #{@uuid}"
+  expect(rows[0][:class]).to include 'current'
+end
+
+Given 'only the first-level Archival Objects are displayed' do
+  rows = all('#tree-container .table .table-row')
+
+  expect(rows.length).to eq 2
+  expect(rows[1].text).to include "Archival Object 1 #{@uuid}"
+end
+
+Then 'the button has text {string}' do |button_text|
+  expect(page).to have_text button_text
+end
+
+Then 'the expand arrows are disabled' do
```

Comment:
similar nitpicky comment - this doesn't actually test that the arrows are disabled only that they have a disabled class

Reply:
Same as above. Probably we should remove this step.

Reply:
I think that it is useful to test for the expected system behavior that the "expand arrows" are disabled. It is similar to the thread about testing for the `.current` CSS class. This may not be the best way, but it meets a need I think.

Reply:
@blacksmith-welder maybe the arrows should just be clicked, then expect that nothing changed (via css selectors, or via children text under an expand arrows are still visible, etc.)

Reply:
Done